cfme/services/catalogs/catalog_item.py

```diff
@@ -140,273 +213,276 @@ def create(self):
         if self.orch_template:
```

Comment:
**Optional**
Would really like to see this as a ternary, possibly even just stuck into the fill dictionary directly since it isn't reused.

``` orch_template = self.orch_template.template_name if self.orch_template else None``